package.json

```diff
@@ -225,6 +225,12 @@
           "description": "Action performed when detected Quarkus properties have an incorrect language.",
           "scope": "window"
         },
+        "qute.server.vmargs": {
```

Comment:
Also, don't forget to update https://github.com/redhat-developer/vscode-quarkus/blob/master/docs/qute/README.md#settings .